website/tutorial/managing-your-application-resources.md

```diff
@@ -0,0 +1,198 @@
+---
+sidebar_position: 6
+title: Managing your application resources
+description: Cover useful actions available to manage your application resources
+keywords: [podman desktop, podman, pod, Containerfile, Dockerfile]
+tags: [podman-desktop, managing-resources, managing-kubernetes-resources]
+---
+
+# Managing your application resources
+
+This tutorial covers the following tasks that you can perform to manage your application resources visually:
+
+- Managing containers and pods
+- Managing images
+- Managing volumes
+- Managing Kubernetes objects
+- Managing other resources
+
+Several useful actions are available in the UI to make resource management easy.
+
+## Before you begin
+
+- [Installed Podman Desktop application](/docs/installation).
+- [A Podman machine](/docs/podman/creating-a-podman-machine).
+- [A valid Kubernetes context and connection](/docs/kubernetes/viewing-and-selecting-current-kubernetes-context) to manage Kubernetes resources.
+
+## Managing containers and pods
+
+Based on your needs, perform any of the following actions after navigating to the component page.
+
+**_Common actions for containers and pods_**
+
+![common actions](img/common-actions.png)
+
+1. Search a container or pod using the search box.
+2. Use the **Start** icon to start a container or pod.
+3. Use the **Stop** icon to stop a container or pod.
+4. Use the **Delete** icon to delete a container or pod.
+
+   :::note
+
+   - You cannot delete a container if it is being used in any pods. In such cases, you must delete the pod and then delete the container.
+
+   :::
+
+5. Bulk deletion: Select containers or pods and delete them by using the **Delete** button.
+6. Click the name of the container or pod to:
+   - View its summary and logs.
+   - View its kube configuration if the container or pod is a Kubernetes resource.
+   - Interact with the container using a terminal
+7. View containers or pods based on their status:
+
+   - Select the **Running** tab to view all running containers or pods
+   - Select the **Stopped** tab to view all stopped containers or pods
+
+8. Use the **Prune** button to remove all unused containers or pods from the Podman engine.
+
+**_Actions specific to containers_**
```

Comment:
Instead of "Actions specific to" for the bolded points, may be easier to read if it's the opposite and it's:


"Container specific actions" so we know exactly what it is when first reading it.